Vikta-API-testing-sample/src/test/java/com/test/api/category/CategoryTest.java

```diff
@@ -62,6 +77,170 @@ public void checkCategoryInAllCategoriesList() {
                 hasItem(equalTo(expectedCategory)));
     }
 
+    /**
+     *  Test with entity search
+     */
+    // city is empty list
+    @Test
+    public void checkIfCanFindEntity(){
+        final String termQueryParam = "term";
+        final String term = "Wolf%20N8%20%2F%20Dynamic%20Marketing%20Officer";
```

Comment:
please list the available categories first and set such search term that would return more than one category. I copied the example from 'Try it out' in the docs and it's working